apps/agora/app/project.json

```diff
@@ -45,11 +45,7 @@
             "output": ""
           }
         ],
-        "styles": [
-          "apps/agora/app/src/styles.scss",
-          "node_modules/primeicons/primeicons.css",
-          "node_modules/primeng/resources/primeng.min.css"
```

Comment:
primeng.min.css was removed since this no longer exists

Reply:
Yes, thank you.  I removed the primeng.min.css references in agora and model-ad as this was breaking the storybook build.